README.md

```diff
@@ -47,9 +49,22 @@ The samples require the following requirements to run.
         <td align="left">UserEmail</td>
         <td align="left">UserEmail of the Admin in your Bold BI, which would be used to get the dashboard list.</td>
     </tr>
+    <tr>
+        <td align="left">DashboardId</td>
+        <td align="left">Item id of the dashboard to be embedded in your application.</td>
+    </tr>
+    <tr>
+        <td align="left">EmbedType</td>
+        <td align="left">BoldBI.EmbedType.Component.</td>
+    </tr>
+    <tr>
+        <td align="left">ExpirationTime</td>
+        <td align="left">Token expiration time.</td>
+    </tr>
   </tbody>
 </table>
 
+ * Run the application.
```

Comment:
Finally this is what I have got recommendation from tool. 

Overall, the steps seem to be clear and straightforward. However, there are a few points that I'd like to expand upon:

1. It would be helpful to provide a link to the ASP.NET Core sample, so that readers can easily find and open it in Visual Studio Code.

2. The wording of the second step is a bit confusing. It's not clear what "able to download" means. Instead, it might be better to say something like "Download the embedConfig.json file from this link".

3. In step 3, it would be helpful to specify where exactly the embedConfig.json file should be pasted in the sample application. For example, should it be placed in the root folder of the project?

4. Step 4 provides a good overview of the properties that are used in the embedConfig.json file. However, it might be helpful to explain what each of these properties does, and why it is necessary to specify them. This would provide more context for readers who are not familiar with Bold BI.

Other than these points, the steps seem to be clear and easy to follow. Good luck with your Bold BI integration!